src/main/java/rookiesspring/controller/ProductController.java

```diff
@@ -19,25 +24,31 @@
 import org.springframework.web.bind.annotation.RestController;
 import rookiesspring.dto.ImageDTO;
 import rookiesspring.dto.ProductDTO;
+import rookiesspring.dto.ProductDTO_1;
+import rookiesspring.dto.UploadImageDTO;
 import rookiesspring.dto.update.ProductUpdateDTO;
+import rookiesspring.service.ImageService;
 import rookiesspring.service.ProductService;
+import rookiesspring.util.Util;
 
 /**
  *
  * @author HP
  */
 @RestController
-@RequestMapping("/product")
+@RequestMapping("/api/v1/products")
 public class ProductController {
 
     private ProductService service;
+    @Autowired
+    private ImageService imageService;
 
     public ProductController(ProductService service) {
         this.service = service;
     }
 
     @GetMapping({"", "/", "/all"})
-    public ResponseEntity getAllProduct( @RequestParam(required = false, value = "name") String name,
```

Comment:
Please help to update APIs I mentioned before to follow the restful API standard. Eg: /all, /full